executor/runtime/docker/docker_ssh.go

```diff
@@ -140,6 +148,9 @@ func addContainerSSHDConfigWithData(c runtimeTypes.Container, tw *tar.Writer, cf
 		}
 		if containerEnv != "" {
 			lines = append(lines, fmt.Sprintf("~v3:titus:%s:%s:%s:%s:%s:%s", username, c.AppName(), containerEnv, c.TaskID(), c.JobGroupStack(), c.JobGroupDetail()))
+			if sshPolicyHash != "" {
+				lines = append(lines, fmt.Sprintf("~v3:instance:%s:%s:%s", username, c.TaskID(), sshPolicyHash))
```

Comment:
If this becomes widespread, we might want something to make it easier for you and me to debug, like:
    
    policyComment := fmt.Sprintf("# This next principal allows %s", sshPolicy)
    policyPrincipal :=  fmt.Sprintf("~v3:instance:%s:%s:%s", username, c.TaskID(), sshPolicyHash)
    lines = append(lines, policyComment, policyPrincipal)

Reply:
Fair enough. I'll make that update.